.github/workflows/e2e.yaml

```diff
@@ -14,16 +14,19 @@ jobs:
     if: ${{ ! contains(github.event.pull_request.labels.*.name, 'ci/skip-e2e') }}
     runs-on: ubuntu-20.04
     strategy:
-      max-parallel: 5 # len(k8sVersion)/2 is a good number to have here
+      max-parallel: 3 # len(k8sVersion)/2 is a good number to have here
       matrix:
         # Latest patch version can be found in https://github.com/kubernetes/website/blob/main/content/en/releases/patch-releases.md
         # Some versions might not be available yet in https://storage.googleapis.com/kubernetes-release/release/v1.X.Y/bin/linux/amd64/kubelet
-        k8sVersion: [ "v1.22.3", "v1.21.4", "v1.20.10", "v1.19.14", "v1.18.20", "v1.17.17", "v1.16.15" ]
+        k8sVersion: [ "v1.24.1","v1.23.8","v1.22.11", "v1.21.14", "v1.16.15" ]
         cri: [ docker ]
+        exclude:
+          - k8sVersion: v1.24.1
+            cri: docker
```

Comment:
not sure why but some container related metrics where not appearing with docker. 
```
ERRO[0273] finding Metric: k8s.container.fsInodesUsed
ERRO[0273] finding Metric: k8s.container.memoryUsedBytes
ERRO[0273] finding Metric: k8s.container.memoryUtilization
ERRO[0273] finding Metric: k8s.container.requestedMemoryUtilization
ERRO[0273] finding Metric: k8s.container.memoryMappedFileBytes
ERRO[0273] finding Metric: k8s.container.oomEventsDelta
ERRO[0273] finding Metric: k8s.container.cpuCfsPeriodsDelta
ERRO[0273] finding Metric: k8s.container.cpuCfsThrottledPeriodsDelta
ERRO[0273] finding Metric: k8s.container.cpuCfsThrottledSecondsDelta
ERRO[0273] finding Metric: k8s.container.cpuCfsPeriodsTotal
ERRO[0273] finding Metric: k8s.container.cpuCfsThrottledPeriodsTotal
ERRO[0273] finding Metric: k8s.container.cpuCfsThrottledSecondsTotal
```

I think @kang-makes had dig into this